UI/MVUX/MVUX/MVUX/Presentation/FeedViewSample/FeedViewModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Uno.Extensions.Reactive;
+
+namespace MVUX.Presentation.FeedViewSample;
+
+public partial record FeedViewModel
+{
+	private readonly Random _random = new();
+	
+	public IFeed<Person> DefaultPerson { get; }
+	
+	public IFeed<Person> CustomLoadingPerson { get; }
+	
+	public IFeed<Person> ErrorPerson { get; }
+	
+	public FeedViewModel()
+	{
+		DefaultPerson = Feed.Async(GetRandomPersonAsync);
+		
+		CustomLoadingPerson = Feed.Async(GetRandomPersonAsync);
```

Comment:
Maybe we could have a separate method for `CustomLoadingPerson` where it takes more time to load `await Task.Delay(10000, ct);` so that the `ProgressTemplate` can be more easily seen.